torch/testing/_internal/opinfo/core.py

```diff
@@ -1437,15 +1442,17 @@ def supported_backward_dtypes(self, device_type):
     def supports_dtype(self, dtype, device_type) -> bool:
         return dtype in self.supported_dtypes(device_type)
 
+    @property
+    def full_name(self):
+        """Returns a full name that helps to uniquely identify this OpInfo."""
+        variant = "." + self.variant_test_name if self.variant_test_name else ""
+        # example: "normal.in_place" where "normal" is the name and "in_place" is the variant
+        return f"{self.name}{variant}"
```

Comment:
small thing but I really want to specify ops by their full name of `"{name}.{variant}"`. `formatted_name` wasn't quite right because it uses underscores instead of periods